### PR TITLE
Adds Loincloths

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Adds Leather and Cloth Loincloths, both craftable from their namesake materials; also adds sensor variants of each available in the Autodrobe and Loadout
Credit to Kroto for the sprites

## Why It's Good For The Game
New craftable clothes - I'm sure ashwalkers will love them dearly

## Changelog
:cl:
add: Added craftable Cloth and Leather loincloths
add: Added sensor-enabled "synthetic" Cloth and Leather loincloths, available in the loadout and Autodrobe
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
oh hey new errors that werent there before, perfect

**Comment by ghost** [comment]
cloth loincloth crafting will have to wait until i sort out why it breaks so badly ;-;

**Comment by Gandalf2k15** [comment]
I love it.

File: modular_skyrat/modules/customization/modules/clothing/under/costumes.dm [hunk 1]
```diff
@@ -59,3 +59,30 @@
...
+/obj/item/clothing/under/costume/loincloth
+	icon = 'modular_skyrat/modules/customization/icons/obj/clothing/uniforms.dmi'
+	worn_icon = 'modular_skyrat/modules/customization/icons/mob/clothing/uniform.dmi'
+	name = "loincloth"
+	desc = "A simple leather covering. It's better than wearing nothing at least."
+	icon_state = "loincloth"
+	body_parts_covered = GROIN
+	can_adjust = FALSE
+	mutant_variants = NONE
+	has_sensor = NONE
```
Comment (KathrinBailey) [inline modular_skyrat/modules/customization/modules/clothing/under/costumes.dm]:
```suggestion
	has_sensor = NO_SENSORS
```

File: modular_skyrat/modules/customization/modules/clothing/under/costumes.dm [hunk 2]
```diff
@@ -59,3 +59,30 @@
...
+
+/obj/item/clothing/under/costume/loincloth/sensor
+	name = "loincloth"
+	desc = "A synthetic leather covering. It's better than wearing nothing at least."
+	has_sensor = TRUE
```
Comment (KathrinBailey) [inline modular_skyrat/modules/customization/modules/clothing/under/costumes.dm]:
```suggestion
	has_sensor = HAS_SENSORS
```

File: modular_skyrat/modules/customization/modules/clothing/under/costumes.dm [hunk 3]
```diff
@@ -59,3 +59,30 @@
...
+
+/obj/item/clothing/under/costume/loincloth/cloth
+	name = "loincloth"
+	desc = "A simple cloth covering. It's better than wearing nothing at least."
+	icon_state = "loincloth_cloth"
+
+/obj/item/clothing/under/costume/loincloth/cloth/sensor
+	name = "loincloth"
+	desc = "A synthetic cloth covering. It's better than wearing nothing at least."
+	has_sensor = TRUE
```
Comment (KathrinBailey) [inline modular_skyrat/modules/customization/modules/clothing/under/costumes.dm]:
```suggestion
	has_sensor = HAS_SENSORS
```

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
Please change the has_sensor variables to use the appropriate defines.

Also, as more of a suggestion **(optional),** you should maybe add a version that covers your butt a bit as well. Kinda like a skirt without the sides. That'd be the main sprite, and you'd then use `spritename_d` in the .dmi and set `can_adjust to TRUE`. That way alt-clicking would toggle if you want a more full loincloth or if you really just want to only cover your junk from the front.

**Ping Floof Ball#0798 once you're done and I'll take another look.**

**Comment by NullDagaf** [comment]
> Also, as more of a suggestion **(optional),** you should maybe add a version that covers your butt a bit as well.

Loincloths don't cover you butt though..? Or am I just dumb.

Edit: Also, if it does, it'd probably be pain for tailed species.

**Comment by ghost** [comment]
> Loincloths don't cover you butt though..? Or am I just dumb.
> 
> Edit: Also, if it does, it'd probably be pain for tailed species.

The loincloth can be less wedgied at least, which is entirely possible yes..
Also the tail goes over the uniform sprite, so it isnt an issue sprite-wise (in fact you cant see the back sprite of the loincloth with most tails so,,)

**Review by KathrinBailey — APPROVED** [review]
Oke dokie

**Comment by NullDagaf** [comment]
> Also the tail goes over the uniform sprite, so it isnt an issue sprite-wise (in fact you cant see the back sprite of the loincloth with most tails so,,)

Meant realistically.

**Pull request merged**